### ISSUE TITLE
JAWS unknow table (Screen reader)

### ISSUE THREAD

**Issue by ittiphol-of** [spipu/html2pdf#279]
Hi everybody,

I created simple table and generated to PDF, then download and open with Acrobat reader then use JAWS to read whole content.

JAWS could read text inside table normally, but cannot see the content table as table ( when I use "T" key ) Jaws or screen reader keep saying "No table found".

Anybody could suggest me please.

=== Example code ====
<h3>Test basic PDF HTML2PDF</h3>
<table>
		<thead>
			<tr><th scope="col">Head1</th><th scope="col">Head2</th><th>Head3</th><th>Head4</th></tr>
		</thead>
		<tbody>
			<tr><td>Items1</td><td>Item2</td><td>Items3</td><td>Item4</td></tr>
			<tr><td>Items5</td><td>Item6</td><td>Items7</td><td>Item8</td></tr>
		</tbody>
</table>
<p>Test text something</p>

===== End example ====

Thank you in advance.

**Comment by spipu** [comment]
because it is not tables, but independent cells.
HTML2PDF does not support this

**Comment by ittiphol-of** [comment]
So, any idea how can I fixed it, on my html template or should I modify HTML2PDF core.

Thank you.
